packages/gateway/test/api.spec.ts

```diff
@@ -35,7 +35,7 @@ import {
   withTransferDomain,
 } from '../src/handlers'
 import { InMemoryRepository } from '../src/repositories'
-import { withSigner, makeMessageHash, withLogger } from '../src/middlewares'
```

Comment:
Why removing Logger middleware?

Reply:
not needed for testing purpose

Reply:
Hmm, the logger is implemented to print thinks on the server when it is already running in production. Got it.